### PR TITLE
set Alert source link to Mimir alerting UI

### PR DESCRIPTION
**Pull request by TheoBrigitte**
Towards: https://github.com/giantswarm/giantswarm/issues/30963

Point Alert source link to Mimir alerting UI when Mimir is enabled

### Alert with the new link

image

### URL opened with the new link

image

### CODE REVIEW

File: service/controller/resource/alerting/alertmanagerconfig/test/notification-template/mimir-enabled/capi/case-1-capa-mc.golden [hunk 1]
```diff
@@ -43,7 +42,7 @@
 {{ if (index .Alerts 0).Annotations.dashboard -}}
 📈 Dashboard: {{ template "__dashboardurl" . }}
 {{ end -}}
-👀 Prometheus: {{ template "__prometheusurl" . }}
+👀 Query: {{ template "__prometheusurl" . }}
```
Comment (QuentinBisson) [inline service/controller/resource/alerting/alertmanagerconfig/test/notification-template/mimir-enabled/capi/case-1-capa-mc.golden:45]: Maybe we should rename __prometheusurl to __queryurl

**Comment by QuentinBisson** [comment]
This is really nice

**Review by QuentinBisson — APPROVED** [review]
My comment is just a nitpick

File: service/controller/resource/alerting/alertmanagerconfig/test/notification-template/mimir-enabled/capi/case-1-capa-mc.golden (hunk 1 above)
Comment (TheoBrigitte) [reply service/controller/resource/alerting/alertmanagerconfig/test/notification-template/mimir-enabled/capi/case-1-capa-mc.golden:45]: Done

**Pull request merged**